### ISSUE TITLE
ERA001 ruff rule should be skipped even if enabled in ruff config

### ISSUE THREAD

**Issue by ssbarnea** [charliermarsh/ruff-vscode#177]
As `ERA001` rule removes (some!) of the commented code, if you run ruff on file-save, it will have some very undesirable side effects while working on the code-base.

I think that ERA001 should be ignored by the ruff vscode extension, to avoid negatively impacting the developer experience.

**Comment by JonathanPlasse** [comment]
You could use `unfixable=["ERA001"]` in the configuration toml. So it will not fix on save.
But then you can run this in the terminal when you are done with your changes.
```console
ruff check --fixable=ERA001 --fix
```

**Comment by charliermarsh** [comment]
Yeah my suggestion here would be to add it as a dedicated command-line argument in your VS Code `settings.json`, like:

```json
